spec/orbital/request_spec/cvv_indicator_spec.rb

```diff
@@ -0,0 +1,164 @@
+require 'spec_helper'
+
+ActiveMerchant::Billing::Base.mode = :test
+
+describe Killbill::Orbital::PaymentPlugin do
+
+  include ::Killbill::Plugin::ActiveMerchant::RSpec
+
+  before(:each) do
+    @plugin = build_plugin(::Killbill::Orbital::PaymentPlugin, 'orbital')
+    @plugin.start_plugin
+    @call_context = build_call_context
+
+    ::Killbill::Orbital::OrbitalPaymentMethod.delete_all
+    ::Killbill::Orbital::OrbitalResponse.delete_all
+    ::Killbill::Orbital::OrbitalTransaction.delete_all
+  end
+
+  after(:each) do
+    @plugin.stop_plugin
+  end
+
+  context 'cvv indicator spec' do
+    before(:each) do
+      @properties = build_pm_properties(nil, { :cc_number => '5454545454545454' })
+      @pm         = create_payment_method(::Killbill::Orbital::OrbitalPaymentMethod, nil, @call_context.tenant_id, @properties, {})
```

Comment:
Do you need this in these tests? I believe this will call the gateway since the endpoint isn't stubbed at this point.